src/transformers/models/herbert/tokenization_herbert.py

```diff
@@ -58,7 +58,7 @@ class HerbertTokenizer(XLMTokenizer):
     pretrained_init_configuration = PRETRAINED_INIT_CONFIGURATION
     max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
 
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
```

Comment:
Some arguments are passed as positionals when instantiating a slow tokenzier to create the fast one.